docs/en/ingest-management/fleet/air-gapped.asciidoc

```diff
@@ -32,6 +32,43 @@ Set the following property in {kib} to enable air-gapped mode in {fleet}. This a
 xpack.fleet.isAirGapped: true
 ----
 
+[discrete]
+[[air-gapped-pgp-fleet]]
+== Configure {agents} to download a PGP/GPG key from {fleet-server}
+
+Starting from version 8.9.0, when {agent} tries to perform an upgrade, it first verifies the binary signature with the key bundled in the agent. This process has a backup mechanism that will use the key coming from `https://artifacts.elastic.co/GPG-KEY-elastic-agent` instead of the one it already has.
+
+In an air-gapped environment, the agent won't be able to download the remote key and therefore cannot be upgraded. For versions 8.9.0 to 8.10.3, you can resolve this problem following the steps described in the associated link:https://www.elastic.co/guide/en/fleet/8.9/release-notes-8.9.0.html#known-issues-8.9.0[known issue] documentation.
+
+
+Starting in version 8.10.4, you can resolve this problem by configuring {agents} to download the PGP/GPG key from {fleet-server}.
```

Comment:
```suggestion
Starting in version 8.10.4, {agent} will:
- verify the binary signature with the key bundled in the agent - if it doesn't pass
- download the PGP/GPG key from `https://artifacts.elastic.co/GPG-KEY-elastic-agent` and verify it - if it doesn't pass
- download the PGP/GPG key from {fleet-server} and verify it
```

Reply:
Thanks! Added in [f698c5e](https://github.com/elastic/ingest-docs/pull/984/commits/f698c5e0a0d5281f279d95865d0c3dd62019841e)